demo-terraform/aws/serverless-app/dynamodb.tf

```diff
@@ -2,29 +2,37 @@
 
 ## Dynamodb Table
 resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure point-in-time-recovery (PITR) is enabled for AWS DynamoDB tables**
 
**Vulnerability**
AWS DynamoDb tables have point-in-time-recovery (PITR) disabled. This may make disaster recovery operations challenging.
 
**Remediation**
Point-in-time-recovery can be enabled for AWS DynamoDB tables.
 In AWS Console - 
 1. Sign in to the AWS Console go to the Amazon DynamoDB console.
 2. Select to the DynamoDB table, then select the Backups tab.
 3. Click Enable.

source: accurics-